gatsby-node.js

```diff
@@ -44,14 +47,17 @@ const createHomePage = async (createPage) => {
 // TODO: add action for creating article pages
 const createArticles = async (createPage) => {
   const newsResp = await axios.get(`${BASE_URL}https://www.thedp.com/section/news.json`)
+  const mostReadDPResp = await axios.get(`${BASE_URL}https://us-central1-web-services-dp.cloudfunctions.net/dropcap/DP`)
+
   newsResp.data.articles.forEach(article => {
     const { authors } = article
     authors.forEach(async author => await createAutors(createPage, author.slug))
     createPage({
       path: article.slug,
       component: ArticleTemplate,
       context: {
-        article
+        article,
+        mostReadDP: mostReadDPResp.data.result.slice(0, 5)
```

Comment:
add destructuring/default params plus do undefined/null checks. This seems really unsafe. What if data is undefined, or result is undefined?